backend/sources/facebook/connector/src/main/java/co/airy/core/sources/facebook/Connector.java

```diff
@@ -169,7 +168,6 @@ public RequestLoggingIgnorePatterns requestLoggingIgnorePatterns() {
     }
 
     @Bean
-    @ConditionalOnProperty("segment.analytics.enabled")
```

Comment:
I think without this conditional there is no way to disable tracking

Reply:
@pascal-airy [This](https://github.com/airyhq/airy/blob/64c6a59314f0a6496e0804b2c73b743f688893f4/lib/java/tracking/src/main/java/co/airy/tracking/TrackingRequestFilter.java#L18) is the place that ultimately controls whether or not data is sent to Segment. If it's not initialized then the `RouteTracking` beans won't do anything